### PR TITLE
adding route path to invalid route message

### PR DESCRIPTION
**Pull request by jackstine**
# Description

Adds route path to the invalid route message.

# Context

Users are finding it hard to identify now invalid route paths, redirects, and rewrites.

# Checklist:

- [ ] I included a concise, user-facing changelog (for details, see https://github.com/solo-io/go-utils/tree/master/changelogutils) which references the issue that is resolved.
- [ ] If I updated APIs (our protos) or helm values, I ran `make -B install-go-tools generated-code` to ensure there will be no code diff
- [ ] I followed guidelines laid out in the Gloo Edge [contribution guide](https://docs.solo.io/gloo-edge/latest/contributing/)
- [ ] I opened a draft PR or added the work in progress label if my PR is not ready for review
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: projects/gloo/pkg/translator/route_config.go [hunk 1]
```diff
@@ -13,6 +13,7 @@ import (
 	envoy_config_route_v3 "github.com/envoyproxy/go-control-plane/envoy/config/route/v3"
 	envoy_type_matcher_v3 "github.com/envoyproxy/go-control-plane/envoy/type/matcher/v3"
 	"github.com/golang/protobuf/ptypes/wrappers"
+	"github.com/rotisserie/eris"
```
Comment (jenshu) [inline projects/gloo/pkg/translator/route_config.go:16]: nit: this is already imported, one line below this

Comment (nfuden) [reply projects/gloo/pkg/translator/route_config.go:16]: Looks good other than this

Comment (jackstine) [reply projects/gloo/pkg/translator/route_config.go:16]: ooo

**Review by jenshu — APPROVED** [review]

**Review by nfuden — APPROVED** [review]

**Comment by jackstine** [comment]
> 

yeah this error gets triggered if you create an invalid route path, rewrite or redirect.
Much [like this customer.](https://github.com/solo-io/gloo/issues/7715#issuecomment-1404572092)

**Pull request merged**